### PR TITLE
feat: new library Azure Icons (@Youri Tjang's library extended)

### PR DESCRIPTION
**Pull request by excalibot**
A collection of Azure icons, based on and extending the icons from "Azure cloud services icons" by  @YouriTjang.

Submitted by @bothzoli

Items: Key Vault, Application Insights, Azure DevOps, Network Interface, Public IP Address, Disk, Azure logo, Subscription, Resource Group, Blob Storage, Network Security Group, Virtual Network, Active Directory, PS Runbook, App Service, Service Bus Namespace, Virtual Machine, SQL Database, Data Lake

[Installation link](https://excalidraw.com/?addLibrary=https%3A%2F%2Fraw.githubusercontent.com%2Fexcalidraw%2Fexcalidraw-libraries%2Fbothzoli-azure-icons-youri-tjang-s-library-extended%2Flibraries%2Fbothzoli%2Fazure-icons-youri-tjang-s-library-extended.excalidrawlib%3Fraw%3Dtrue)

### CODE REVIEW

**Comment by dwelle** [comment]
Cool! @bothzoli @YouriTjang I suggest we merge these under the existing lib, listing both of you as authors? I can do this later today.

File: libraries.json [hunk 1]
```diff
@@ -1796,5 +1796,22 @@
     "created": "2022-03-25",
     "updated": "2022-03-25",
     "version": 2
+  },
+  {
+    "name": "Azure Icons (@Youri Tjang's library extended)",
+    "description": "A collection of Azure icons, based on and extending the icons from \"Azure cloud services icons\" by @Youri Tjang.",
```
Comment (dwelle) [inline libraries.json:1802]:
```suggestion
    "description": "A collection of Azure icons, based on and extending the icons from \"Azure cloud services icons\" by @YouriTjang.",
```

**Comment by bothzoli** [comment]
Thanks @dwelle! I agree with merging the two :)

**Comment by YouriTjang** [comment]
Nice addition @bothzoli 
Makes sense to merge, thx @dwelle

**Comment by dwelle** [comment]
Sorry for the delay, merging! 🚀

**Pull request merged**